package.json

```diff
@@ -5,7 +5,8 @@
   "workspaces": [
     "packages/*",
     "packages/legacy/core",
-    "packages/legacy/app"
+    "packages/legacy/app",
+    "packages/react-native-attestation/example"
```

Comment:
I don't think we want this added. It's just some janky code that came along with the skeleton / templating when I created the package. I should either remove it or make it an actual thing.

Reply:
if not needed, then we should delete the example folder. It is out of date and the name of the package is incorrect.

Reply:
Ok. I removed it in my PR to fix the publishing. You can remove it here also.